krun/vm_defs.py

```diff
@@ -318,7 +317,7 @@ def __eq__(self, other):
 
     def parse_instr_stderr_file(self, file_handle):
         """Parse instrumentation data. Override as necessary"""
-        pass
+        return {}
```

Comment:
I would only use `dict` if I were building a dict from a list of tuples or something.